src/dev/git-flow.md

```diff
@@ -0,0 +1,91 @@
+# Git Flow
+
+We recommand to use the following git flow four our applications and konnectors.
+
+## Development
+
+`master` is the current state of the development.
+
+Every commit must:
+
+* provide a working application without introducing blocking regressions
+* validate lint and unit tests
+
+Every feature, improvement or fix is merged via a pull request.
+
+For large features with huge impacts on the whole codebase, use feature branches, for example `feature/something-new`. A feature branch is merged into master when it's totally functional and validated.
+
+## Releases
+
+> TL;DR
+>
+> * Create release branch
+> * Bump `master` version
+> * Tag beta versions from release branch until it passes all tests
+> * Tag stable version from release branch
+> * Delete release branch
+
+A new release of an app or a konnector is always based on the current state of `master`, so just create a new branch for the given release version and bump `master` to a new minor version.
+
+Example with `master` being versionned to `1.4.0`:
+```sh
+git checkout master
+git checkout -b release-1.4.0
+git checkout master
+# Update app version to 1.5.0 in package.json and manifest.webapp
+git add package.json manifest.webapp
+git commit -m "chore: Bump version to 1.5.0"
+```
```

Comment:
Several questions here:
- why don't we create a branch without moving to it, ie replace `git checkout master && git checkout -b release-1.4.0 && git checkout master` by `git checkout master && git branch release-1.4.0`.
- don't we want to `git fetch origin && git checkout master && git rebase origin/master`?
- do we want to create the release `1.5.0` or `1.4.0` (see below)? There is confusion.

Reply:
We create the release 1.4.0.
Right after we bump master's version, so next development are being made under a new version.
Then we can go back to our release-1.4.0 branch.

We assume that master is up to date.